src/components/Trashy/TrashySchedule.js

```diff
@@ -33,10 +34,12 @@ const TrashySchedule = props => {
   const pickUpList = pickupDates.map(item => {
     const services = item.services.map(service => {
       let s = mapService(service);
-      return <li>{s}</li>;
+      // check if service was mapped correctly, otherwise s will be undefined 
+      return (s && <li>{s}</li>);
     });
 
     return (
+      !!services.length && services[0] &&
```

Comment:
I always forget that `[undefined]` length is `1`

Reply:
'ol... "undefined" always sneaking a bit of "defined" in!😶